packages/create-mc-app/bin/create-mc-app.js

```diff
@@ -49,6 +55,9 @@ async function execute() {
   console.log('');
   console.log(`To get started:`);
   console.log(`$ cd ${options.projectDirectoryName}`);
+  if (flags['skip-install']) {
```

Comment:
Just wondering. Could also be `auto-install` and default to `true` while CI passes `false` maybe?

Reply:
Hmm I think I find "skip" to be more appropriate, because we are skipping one of the steps/tasks. Thanks for the suggestion though!